doc/roadmap.md

```diff
@@ -32,15 +32,18 @@ Disclaimer:
 The following list of features are planned to be addressed in the short term,
 and incorporated in a next release of the product in the short term:
 
-* *None at the present moment*
+- Performance / scalability / sizing and reliability analisys.
 
 ## Medium term
 
 The following list of features are planned to be addressed in the medium term,
 typically within the subsequent release(s) generated in the next **9 months**
 after next planned release:
 
-* *None at the present moment*
+- Brand new sink able to write to IoTAs and NGSI endpoints.
```

Comment:
All IoTAs or IOTA-JSON specify? In the second case:

```suggestion
- Brand new sink able to write to IoTA JSON and NGSI endpoints.
```

In addition, maybe the NGSI version should be make explicit and use NGSIv2 here.